src/main/markdown/doc/latest/DevGuideI18nLocale.md

```diff
@@ -76,9 +76,13 @@ are probably better off choosing some real language as a default, such as
 the following in your [module XML](DevGuideOrganizingProjects.html#DevGuideModuleXml):
 
 ```
-<set-property-fallback name="locale" value="en"/>
+<extend-property name=locale" value"es,de,fr,en,it" <!-- 1. full list of supported locales --> 
+<set-property-fallback name="locale" value="en"/> <!-- 2. fallback locale -->
+<set-property name="locale" value="es,de,fr,en,it"/> <!-- 3. full list of supported localesm again -->
 ```
 
+Note that there is an open issue related to this that requires us to specify the full list of supported locales _after_ the fallback locale to prevent the inclusion of an unused `default` locale.
```

Comment:
I still think this paragraph should be removed entirely (also, the emphasis on _after_ is wrong, order doesn't matter)